django/test/runner.py

```diff
@@ -795,6 +795,9 @@ def build_suite(self, test_labels=None, extra_tests=None, **kwargs):
             # Since tests are distributed across processes on a per-TestCase
             # basis, there's no need for more processes than TestCases.
             processes = min(self.parallel, len(subsuites))
+            # Update "parallel" because it's used to determine the number of
+            # test databases.
+            self.parallel = processes
```

Comment:
If you're updating `self.parallel` anyway, can you get rid of `processes`?

Then you don't need the comment or the extra assignment…

Reply:
I think it's clearer that way :thinking: . IMO, using the same variable (without any comment) was one of the reasons we introduced this regression.

Reply:
Or you could merge the comment with the one above?

Reply:
I think it's okay to use the variable, IMO, especially since it's used twice further down.

Reply:
OK, updated.

Reply:
Just to be clear, I meant `processes` variable. But if you want to go with the other suggestion, I won't object.

Reply:
Reverted :smile:

Reply:
Thank you. Needless to say, I would say I find your English actually to be quite excellent.🥇